pull.py

```diff
@@ -103,8 +146,41 @@
     'FDL1.2': {'spdx': 'GFDL-1.2'},
     'FDL1.3': {'spdx': 'GFDL-1.3'},
     'FreeBSD': {'spdx': 'BSD-2-Clause-FreeBSD'},
+    'freetype': {'spdx': 'FTL'},
     'GNUAllPermissive': {'spdx': 'FSFAP'},
     'GNUGPLv3': {'spdx': 'GPL-3.0'},
+    'gnuplot': {'spdx': 'gnuplot'},
+    'GPLv2': {'spdx': 'GPL-2.0'},
+    'HPND': {'spdx': 'HPND'},
+    'IBMPL': {'spdx': 'IPL-1.0'},
+    'iMatix': {'spdx': 'iMatix'},
+    'imlib': {'spdx': 'Imlib2'},
+    'ijg': {'spdx': 'IJG'},
+    'intel': {'spdx': 'Intel'},
+    'IPAFONT': {'spdx': 'IPA'},
+    'ISC': {'spdx': 'ISC'},
+    'JSON': {'spdx': 'JSON'},
+    'LAL-1.2': {'spdx':'LAL-1.2'},
+    'LAL-1.3': {'spdx':'LAL-1.3'},
+    'LGPLv3': {'spdx': 'LGPL-3.0'},
+    'LGPLv2.1': {'spdx': 'LGPL-2.1'},
+    'LPPL-1.2': {'spdx': 'LPPL-1.2'},
+    'LPPL-1.3a': {'spdx': 'LPPL-1.3a'},
+    'lucent102': {'spdx': 'LPL-1.02'},
+    'ModifiedBSD': {'spdx': 'BSD-3-Clause'},
+    'MPL': {'spdx': 'MPL-1.1'},
+    'MPL-2.0': {'spdx':'MPL-2.0'},
+    'ms-pl': {'spdx': 'MPL-1.1'},
+    'ms-rl': {'spdx': 'MS-RL'},
+    'NASA': {'spdx': 'NASA-1.3'},
+    'NCSA': {'spdx':'NCSA'},
+    'newOpenLDAP': {'spdx': 'OLDAP-2.7'},
+    'Nokia': {'spdx': 'Nokia'},
+    'NoLicense': {'spdx': 'UNSPECIFIED'},
```

Comment:
I read [the FSF's `NoLicense` entry][1] as "all rights reserved".  [`NOASSERTION`][2] is different.  I expect `NoLicense` has no SPDX equivalent at the moment.

[1]: https://www.gnu.org/licenses/license-list.html#NoLicense
[2]: https://github.com/spdx/spdx-spec/blame/cfa1b9d08903befdf03e669da6472707b7b60cb9/chapters/3-package-information.md#L148

Reply:
Agree - I'll amend the PR